lib/tests/post_iris_call_receive_header.js

```diff
@@ -0,0 +1,63 @@
+const expect = require('chai').expect;
+const { getRskTransactionHelper } = require('../rsk-tx-helper-provider');
+const { getBtcClient } = require('../btc-client-provider');
+const { getBridge, getLatestActiveForkName } = require('../precompiled-abi-forks-util');
+const { activateFork, sendTxWithCheck, getLatestForkName, waitAndUpdateBridge } = require('../rsk-utils');
+const { ensure0x } = require('../utils');
+
+let rskTxHelper;
+let btcTxHelper;
+let bridge;
+
+const fulfillRequirementsToRunAsSingleTestFile = async () => {
+    const forkName = process.env.FORK_NAME || getLatestForkName().name;
+    await activateFork(Runners.common.forks[forkName]);
+};
+
+const execute = (description, getRskHost) => {
+    describe(description, function() {  
+        before(async () => {
+            rskTxHelper = getRskTransactionHelper(getRskHost());
+            btcTxHelper = getBtcClient();
+    
+            if(process.env.RUNNING_SINGLE_TEST_FILE) {
+              await fulfillRequirementsToRunAsSingleTestFile();
+            }
+
+            bridge = getBridge(rskTxHelper.getClient(), await getLatestActiveForkName());
+        });
+    
+        it('should return 0 and increment BTC blockchain size when calling receiveHeader method', async () => {
+            const cowAddress = await rskTxHelper.newAccountWithSeed('cow');
+
+            await waitAndUpdateBridge(rskTxHelper);
+
+            const blockHashes = await btcTxHelper.mine();
+            const blockHeader = await btcTxHelper.getBlockHeader(blockHashes[0], false);
+            const blockchainInitialHeigth = await bridge.methods.getBtcBlockchainBestChainHeight().call();
+
+            await sendTxWithCheck(
```

Comment:
Can we put the method call and the callback in the own variables? It will make it easier to read